src/lib/web-component/log-view/log-view.css

```diff
@@ -19,7 +24,11 @@
 }
 
 .log-details {
-  margin: 50px 15px 10px;
+  margin: 40px 0 10px;
+  white-space: pre-wrap;
+  max-height: 450px;
+  overflow-y: scroll;
```

Comment:
have you checked what happens if the json log contains something (e.g. a very long string value or a very long object property name)? would it has to also scroll horizontally? would setting the overflow to `auto` also work and made the scrollbar hidden when the json fits into the available space?

Reply:
When  there us a log string value, it gets indent into the next time. Have a look at the screenshot.
### screenshot
<img width="521" alt="Screenshot 2020-07-28 at 11 49 08 PM" src="https://user-images.githubusercontent.com/8364578/88702290-f35dd300-d12c-11ea-9d3e-695485161f23.png">

Reply:
> would setting the overflow to `auto` also work and made the scrollbar hidden when the json fits into the available space?

Yeah, I have updated the overflow to be `auto` :-)